docs/Native-framework-changelog.md

```diff
@@ -4,6 +4,10 @@ We regularly update our framework to support new features. If you have an older
 
 ## Latest versions (3.x)
 
+### Version 3.9.1 (December 10, 2018)
+
+- Fixed file download error saving
```

Comment:
@angelnikolov  `Fixed an issue which caused some apps to crash when applying app updates under certain conditions` ;)

Reply:
just because this changelog is public